examples/blog/articles/models.py

```diff
@@ -32,7 +32,7 @@ class NewUser(User):
     new_field = StringField()
 
     def __unicode__(self):
-        return self.email
+        return unicode(self.first_name)
```

Comment:
Why is this changed to `first_name`? It no longer matches the `User` class above. I'd prefer this continue to be the email. 

Also, why is this getting caste to unicode?